pkg/util/util.go

```diff
@@ -38,25 +38,6 @@ func GetSecret(kubeClient client.Client, secretName, namespace string) (*corev1.
 	return s, nil
 }
 
-// NewGCPSecretCR returns a Secret CR formatted for GCP
-// To be removed along with ClusterDeployment controller
-func NewGCPSecretCR(namespace, creds string) *corev1.Secret {
-	return &corev1.Secret{
-		Type: "Opaque",
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "Secret",
-			APIVersion: "v1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      gcpSecretName,
-			Namespace: namespace,
-		},
-		Data: map[string][]byte{
-			"osServiceAccount.json": []byte(creds),
-		},
-	}
-}
-
 // NewGCPSecretCRV2 returns a Secret CR formatted for GCP for use in projectreference controller.
 func NewGCPSecretCRV2(creds string, namespacedNamed kubetypes.NamespacedName) *corev1.Secret {
```

Comment:
Can we rename this function to  NewGCPSecretCR and adjust the comments as well

Reply:
Renamed function and comment